content/events/filecoin-hacker-base.md

```diff
@@ -0,0 +1,9 @@
+---
+eventName: Filecoin Hacker Base
```

Comment:
Is this event different from the filecoin meetup @ ETHDenver? @jenks-guo-filecoin

Reply:
Hey, I believe this is different from the Meetup that PLN or Recruitment is organizing. Hacker Base is going to be a two-days event that might host a few events and be used for co-working and some ad-hoc events(Lightning talks, Tableland Workshop & FVM Content).